connectivity/drivers/ble/FEATURE_BLE/TARGET_STM32WB/HCIDriver.cpp

```diff
@@ -480,10 +483,29 @@ class TransportDriver : public CordioHCITransportDriver {
             WirelessFwInfo_t wireless_info_instance;
             WirelessFwInfo_t *p_wireless_info = &wireless_info_instance;
             if (SHCI_GetWirelessFwInfo(p_wireless_info) != SHCI_Success) {
-                tr_info("SHCI_GetWirelessFwInfo error");
+                tr_error("SHCI_GetWirelessFwInfo error");
             } else {
+                // https://github.com/STMicroelectronics/STM32CubeWB/tree/master/Projects/STM32WB_Copro_Wireless_Binaries
+                // Be sure that you are using the latest BLE FW version
                 tr_info("WIRELESS COPROCESSOR FW VERSION ID = %d.%d.%d", p_wireless_info->VersionMajor, p_wireless_info->VersionMinor, p_wireless_info->VersionSub);
-                tr_info("WIRELESS COPROCESSOR FW STACK TYPE = %d", p_wireless_info->StackType);
+                tr_info("WIRELESS COPROCESSOR FW STACK TYPE = %d (ROM size 0x%x)", p_wireless_info->StackType, MBED_ROM_SIZE);
+
+#if STM32WB55xx
+                switch (p_wireless_info->StackType) {
+                    case INFO_STACK_TYPE_BLE_FULL:
+                        if (MBED_ROM_SIZE > 0xCA000)  {
```

Comment:
This does not depends on the stack version ? The size in ROM is guaranteed to stay the same ?

Reply:
Yes, it size is changing, test would need to add a version condition